cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/w/ugrad.20/cs/zheyuan/CS118Lab1/cs118-project1/CS118-Project1/.waf-1.6.11-e259c754a0d76606fd48525f48505e56/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/w/ugrad.20/cs/zheyuan/CS118Lab1/cs118-project1/CS118-Project1/.waf-1.6.11-e259c754a0d76606fd48525f48505e56/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 34772112: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-fcolor-diagnostics\', \'-DHAVE_STRING_H=1\', \'-DHAVE_MEMMEM=1\', \'-DHAVE_STPNCPY=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
